plenum/server/replica.py

```diff
@@ -2299,8 +2315,8 @@ def dequeue_pre_prepares(self):
 
     def enqueue_prepare(self, pMsg: Prepare, sender: str):
         key = (pMsg.viewNo, pMsg.ppSeqNo)
-        self.logger.info("{} queueing prepare due to unavailability of PRE-PREPARE. "
-                         "Prepare {} for key {} from {}".format(self, pMsg, key, sender))
+        # self.logger.info("{} queueing prepare due to unavailability of PRE-PREPARE. "
```

Comment:
Please revert back to debug log level instead of comment it out

Reply:
ok